### PR TITLE
boards: 96b_carbon: add supported features

### PR DESCRIPTION
**Pull request by nashif**
add i2c, spi and usb_device as supported feaures.

Signed-off-by: Anas Nashif <anas.nashif@intel.com>

### CODE REVIEW

**Review by ydamigos — APPROVED** [review]

File: boards/arm/96b_carbon/96b_carbon.yaml [hunk 1]
```diff
@@ -5,3 +5,7 @@ arch: arm
 toolchain:
   - zephyr
   - gccarmemb
+supported:
+  - i2c
+  - spi
+  - usb_device
```
Comment (erwango) [inline boards/arm/96b_carbon/96b_carbon.yaml]: Shouldn't we add ble (or bluetooth)?

Comment (galak) [reply boards/arm/96b_carbon/96b_carbon.yaml]: Agreed, also gpio, pinmux?

**Review by galak — CHANGES_REQUESTED** [review]
If we are updating this, lets make it complete for netif and other peripherals.

**Comment by nashif** [comment]
add ble, gpio
We have not added pinmux to any boards, kinda assumed. not sure what you want to put under netif, this thing does not have a networking adapter. if you mean ble, we have not done this for any board with ble support.

Anyways, all of this we need to start getting from dts, waiting for critical mass and additional things supported in dts then we can look at it again.

**Review by galak — APPROVED** [review]

**Pull request merged**